Archives/VolFile.cpp

```diff
@@ -506,4 +503,8 @@ namespace Archives
 		}
 		m_NumberOfPackedFiles = packedFileCount;
 	}
+
+	VolFile::SectionHeader::SectionHeader() {}
+	VolFile::SectionHeader::SectionHeader(std::array<char, 4> tag, uint32_t length, VolPadding padding) 
+		: tag(tag), length(length), padding(padding) {}
```

Comment:
Good, though I'm not sure if this needs to be explicit. Public struct fields can be initialized with brace syntax. There may be some benefits to using constructor notation though. Maybe experiment with using `= default`, at least for the empty constructor.